src/main/java/entities/User.java

```diff
@@ -81,6 +79,9 @@ private List<Object> extractReviewBody(Review review){
         return revBody;
```

Comment:
I wonder if you can put extractReviewBody method into Reviews class

Reply:
Sorry, it is not written by me. I have no idea about the  Review use case.